src/components/wp-version-selector/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { SelectControl, Icon } from '@wordpress/components';
```

Comment:
Now we have separate component which doesn't store any context, so it's already used in 2 places and can be easily reused in the future, w/o extra conditions.